internal/pkg/flink/internal/controller/input_controller_utils.go

```diff
@@ -40,3 +43,23 @@ func restoreStdin(state *term.State) {
 		_ = term.Restore(int(os.Stdin.Fd()), state)
 	}
 }
+
+func outputErr(s string) {
+	c := color.New(config.ErrorColor)
+	output.Println(c.Sprintf(s))
+}
+
+func outputInfo(s string) {
+	c := color.New(config.InfoColor)
+	output.Println(c.Sprint(s))
+}
+
+func outputInfof(s string, args ...any) {
+	c := color.New(config.InfoColor)
+	output.Printf(c.Sprint(s), args...)
+}
+
+func outputWarn(s string) {
+	c := color.New(config.WarnColor)
+	output.Println(c.Sprint(s))
+}
```

Comment:
Can you integrate this with the existing CLI logger? We have support for INFO/DEBUG/WARN/ERROR, which is filtered with the `-v` flags.

Reply:
For now, these functions could just call the CLI logger functions

Reply:
It makes sense to have a central cli logger but I don't think it makes sense merging these two tho.

The thing here is, even though I named this "info", "warning" and "errors", there's are no use cases where the user don't want to see all infos these. It was mostly about coloring here and not level of importance. All these infos are essential for the SQL Client user. See the image below:

![image](https://github.com/confluentinc/cli/assets/11739405/b21c939a-d234-45e8-a7e9-206a664bdc3a)

The user needs always need all information in white, red and yellow.

Also, even if we somehow do think it makes sense to merge these, I wouldn't want to do this as part of this PR/part of this jira ticket, which is only related to adding different colors to different types of message. I would prefert to do this in interactive steps and create another ticket :)